app/build.gradle.kts

```diff
@@ -0,0 +1,226 @@
+import java.util.*
+
+plugins {
+    id("com.android.application")
+    id("kotlin-android")
+    id("kotlin-kapt")
+    id("kotlin-parcelize")
+    id("androidx.navigation.safeargs.kotlin")
+    id("realm-android")
+    id("io.sentry.android.gradle")
+    id("de.mannodermaus.android-junit5")
+}
+
+android {
+
+    namespace = "com.infomaniak.drive"
+
+    compileSdk = 34
+
+    ndkVersion = "28.0.13004108"
+
+    defaultConfig {
+        testInstrumentationRunnerArguments["runnerBuilder"] = "de.mannodermaus.junit5.AndroidJUnit5Builder"
+        applicationId = "com.infomaniak.drive"
+        minSdk = 24
+        targetSdk = 34
+        versionCode = 5_05_000_01
+        versionName = "5.5.0"
+
+        setProperty("archivesBaseName", "kdrive-$versionName ($versionCode)")
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        buildConfigField("String", "CLIENT_ID", """"9473D73C-C20F-4971-9E10-D957C563FA68"""")
+        buildConfigField("String", "CREATE_ACCOUNT_URL", """"https://welcome.infomaniak.com/signup/ikdrive?app=true"""")
+        buildConfigField("String", "CREATE_ACCOUNT_SUCCESS_HOST", """"kdrive.infomaniak.com"""")
+        buildConfigField("String", "CREATE_ACCOUNT_CANCEL_HOST", """"welcome.infomaniak.com"""")
+        buildConfigField("String", "DRIVE_API_V1", """"https://api.kdrive.infomaniak.com/drive/"""")
+        buildConfigField("String", "DRIVE_API_V2", """"https://api.kdrive.infomaniak.com/2/drive/"""")
+        buildConfigField("String", "DRIVE_API_V3", """"https://api.kdrive.infomaniak.com/3/drive/"""")
+        buildConfigField("String", "MANAGER_URL", """"https://manager.infomaniak.com/v3/"""")
+        buildConfigField("String", "OFFICE_URL", """"https://kdrive.infomaniak.com/app/office/"""")
+        buildConfigField("String", "SHARE_URL_V1", """"https://kdrive.infomaniak.com/app/"""")
+        buildConfigField("String", "SHARE_URL_V2", """"https://kdrive.infomaniak.com/2/app/"""")
+        buildConfigField("String", "SHARE_URL_V3", """"https://kdrive.infomaniak.com/3/app/"""")
+        buildConfigField("String", "SHOP_URL", """"https://shop.infomaniak.com/order/"""")
+        buildConfigField("String", "SUPPORT_URL", """"https://support.infomaniak.com/"""")
+
+        resValue("string", "CLOUD_STORAGE_AUTHORITY", "com.infomaniak.drive.documents")
+        resValue("string", "FILE_AUTHORITY", "com.infomaniak.drive.files")
+
+        resValue("string", "EXPOSED_UPLOAD_DIR", "upload_media")
+        resValue("string", "EXPOSED_OFFLINE_DIR", "offline_storage")
+        resValue("string", "EXPOSED_PUBLIC_SHARE_DIR", "public_share")
+
+        androidResources.localeFilters += arrayOf("en", "de", "es", "fr", "it")
```

Comment:
There is a strange warning on this line.

Reply:
It's not a strange warning, it's that the API is incubating, which is Gradle terms for "we might change it a bit, not 100% stable API yet".
I think it's better than the deprecated equivalent that we were using.